benchmarks/pom.xml

```diff
@@ -45,6 +45,13 @@
       <groupId>com.squareup.moshi</groupId>
       <artifactId>moshi</artifactId>
       <version>1.11.0</version>
+      <exclusions>
```

Comment:
did this break something or was it merge drift?

Reply:
https://github.com/openzipkin/zipkin/pull/3236#discussion_r505201557
yeah, it breaks the build.

Reply:
thx!